src/Type/Accessory/AccessoryNumericStringType.php

```diff
@@ -78,6 +78,10 @@ public function isSubTypeOf(Type $otherType): TrinaryLogic
 
 	public function isAcceptedBy(Type $acceptingType, bool $strictTypes): TrinaryLogic
 	{
+		if ($acceptingType->isNonEmptyString()->yes()) {
```

Comment:
I tried adding this case in isSubTypeOf() instead, but this made other tests fail. therefore the fix looks like it is right now.